pubpub/platform
# Update google drive action

## Issue(s) Resolved

## High-level Explanation of PR
- Casts some values so they validate
- Allows use of set communitySlug rather than hardcoding
- Adds discussions and versions only when they don't already exist
- Adds a new version if old and new content don't match

## Test Plan
1. Test the google drive action on the gdrive folder with a URL field configured to hold a gdrive folder
2. Ensure that:
- On a first run, new discussions and versions are created
- On second run, no new versions and discussions are created
3. Update the main doc in the google drive folder
4. Re-run the action
5. Ensure that a new version is created, with new content

## Screenshots (if applicable)

## Notes
